dependencies/third-party/crc32/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.0)
+project(crc32)
+
+if (NOT WIN32) 
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fpic")
```

Comment:
You could use the CMake way, which is:
`set_property(TARGET ${PROJECT_NAME} PROPERTY POSITION_INDEPENDENT_CODE ON)`
but this call should be made after
`add_library()`